addOns/automation/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Add support for the wait time of the Client Script Based Authentication.
```

Comment:
...configuration support....time after Client Script Based Authentication.

Reply:
I will need the whole sentence.

Reply:
No problem. I was thinking:
- Add configuration support for the wait time after Client Script Based Authentication.